src/test/script_tests.cpp

```diff
@@ -1,18 +1,20 @@
+// Copyright (c) 2011-2020 The Bitcoin Core developers
```

Comment:
Should probably go to the current year
```suggestion
// Copyright (c) 2011-2021 The Bitcoin Core developers
```

Reply:
This change contains no upstream changes from 2021.  We only update the header if it was changed by the authors in that year.